### PR TITLE
Update ouroboros-network and cardano-ledger dependencies

### PR DESCRIPTION
**Pull request by coot**
The PR is supposed to update `cardano-node` with a newer version of `ouroboros-network` (and `cardano-ledger`).

The networking team decided to [not release P2P](https://github.com/input-output-hk/ouroboros-network/pull/3975) on `NodeToNodeV_10` and to avoid broken `cardano-node` (which would happen if `Conway` was released with incompatible version of `NodeToNodeV_10`).

This PR subsumes #4359.

### CODE REVIEW

**Review by JurbinM — APPROVED** [review]

**Comment by coot** [comment]
`ouroboros-network` was already updated passed input-output-hk/ouroboros-network@9128bfcc0

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]
Yet again due to a lack of communication we have duplicated work: https://github.com/input-output-hk/cardano-node/pull/4381

File: cardano-api/src/Cardano/Api/LedgerEvent.hs [hunk 1]
```diff
@@ -52,6 +45,12 @@ import           Ouroboros.Consensus.Ledger.Basics (AuxLedgerEvent)
 import           Ouroboros.Consensus.Shelley.Ledger (ShelleyBlock,
                    ShelleyLedgerEvent (ShelleyLedgerEventTICK))
 import           Ouroboros.Consensus.TypeFamilyWrappers
+import Cardano.Ledger.Shelley.Rules.Tick (ShelleyTickEvent(..))
```
Comment (Jimbo4350) [inline cardano-api/src/Cardano/Api/LedgerEvent.hs:48]: These imports do not follow the style convention

File: cardano-api/src/Cardano/Api/LedgerState.hs [hunk 2]
```diff
@@ -167,6 +167,8 @@ import qualified Ouroboros.Network.Block
 import qualified Ouroboros.Network.Protocol.ChainSync.Client as CS
 import qualified Ouroboros.Network.Protocol.ChainSync.ClientPipelined as CSP
 import           Ouroboros.Network.Protocol.ChainSync.PipelineDecision
+import qualified Cardano.Ledger.Shelley.API as ShelleyAPI
```
Comment (Jimbo4350) [inline cardano-api/src/Cardano/Api/LedgerState.hs:170]: These imports do not follow the style convention

File: cardano-api/src/Cardano/Api/Orphans.hs [hunk 3]
```diff
@@ -10,73 +10,74 @@
...
-import           Cardano.Slotting.Slot (SlotNo (..))
-import           Cardano.Slotting.Time (SystemStart (..))
-import           Control.State.Transition (STS (State))
-
+import Cardano.Api.Script
 import qualified Cardano.Binary as CBOR
 import qualified Cardano.Crypto.Hash.Class as Crypto
 import qualified Cardano.Ledger.Alonzo.Data as Alonzo
-import qualified Cardano.Ledger.Alonzo.Scripts as Alonzo
+import Cardano.Ledger.Babbage (BabbageTxOut)
```
Comment (Jimbo4350) [inline cardano-api/src/Cardano/Api/Orphans.hs:22]: These imports do not follow the style convention

File: cardano-api/src/Cardano/Api/Orphans.hs [hunk 4]
```diff
@@ -101,233 +102,265 @@ instance ToJSON Shelley.AccountState where
...
   ]
 
-instance forall era.
-         ( Consensus.ShelleyBasedEra era
-         , ToJSON (Core.TxOut era)
-         , ToJSON (Core.PParams era)
-         , ToJSON (Core.PParamsDelta era)
-         ) => ToJSON (Shelley.EpochState era) where
+instance
+  forall era.
```
Comment (Jimbo4350) [inline cardano-api/src/Cardano/Api/Orphans.hs:110]: Why have you modified the style here?

File: cardano-api/src/Cardano/Api/Orphans.hs [hunk 5]
```diff
@@ -101,233 +102,265 @@ instance ToJSON Shelley.AccountState where
...
+  ) =>
+  ToJSON (Shelley.EpochState era)
+  where
   toJSON = object . toEpochStatePairs
   toEncoding = Aeson.pairs . mconcat . toEpochStatePairs
 
 toEpochStatePairs ::
-  ( Consensus.ShelleyBasedEra era
-  , ToJSON (Core.TxOut era)
-  , ToJSON (Core.PParamsDelta era)
```
Comment (Jimbo4350) [inline cardano-api/src/Cardano/Api/Orphans.hs:120]: Why have you modified the style here?

File: cardano-api/src/Cardano/Api/Orphans.hs [hunk 6]
```diff
@@ -101,233 +102,265 @@ instance ToJSON Shelley.AccountState where
...
+  [a]
 toEpochStatePairs eState =
   let !esAccountState = Shelley.esAccountState eState
       !esSnapshots = Shelley.esSnapshots eState
       !esLState = Shelley.esLState eState
       !esPrevPp = Shelley.esPrevPp eState
       !esPp = Shelley.esPp eState
       !esNonMyopic = Shelley.esNonMyopic eState
-  in  [ "esAccountState" .= esAccountState
-      , "esSnapshots" .= esSnapshots
```
Comment (Jimbo4350) [inline cardano-api/src/Cardano/Api/Orphans.hs:134]: Why have you modified the style here?

File: cardano-api/src/Cardano/Api/Orphans.hs [hunk 7]
```diff
@@ -101,233 +102,265 @@ instance ToJSON Shelley.AccountState where
...
+        "esSnapshots" .= esSnapshots,
+        "esLState" .= esLState,
+        "esPrevPp" .= esPrevPp,
+        "esPp" .= esPp,
+        "esNonMyopic" .= esNonMyopic
       ]
 
-
-instance ( Consensus.ShelleyBasedEra era
-         , ToJSON (Core.TxOut era)
```
Comment (Jimbo4350) [inline cardano-api/src/Cardano/Api/Orphans.hs:143]: Why have you modified the style here?

File: cardano-api/src/Cardano/Api/Orphans.hs [hunk 8]
```diff
@@ -101,233 +102,265 @@ instance ToJSON Shelley.AccountState where
...
+    ToJSON (Core.PParamsUpdate era)
+  ) =>
+  ToJSON (Shelley.LedgerState era)
+  where
   toJSON = object . toLedgerStatePairs
   toEncoding = Aeson.pairs . mconcat . toLedgerStatePairs
 
 toLedgerStatePairs ::
-  ( Consensus.ShelleyBasedEra era
-  , ToJSON (Core.TxOut era)
```
Comment (Jimbo4350) [inline cardano-api/src/Cardano/Api/Orphans.hs:151]: Why have you modified the style here?

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]
I haven't reviewed the PR put please fix all of the imports to follow the style of the library and undo any style changes

**Comment by Soupstraw** [comment]
closed in favor of #4608

**Pull request closed without merging**